python/tvm/build_module.py

```diff
@@ -52,17 +57,58 @@ def __getattr__(self, name):
             return BuildConfig.defaults[name]
         return self._attr[name]
 
+    def decorator(self, func):
+        ''' decorate the pass function'''
+        def dump(*args, **kwargs):
+            '''dump function'''
+            retv = func(*args, **kwargs)
+            # pylint: disable=unidiomatic-typecheck
```

Comment:
do not remove pylint checks unless you think is necessary. In this case, it tells you to use instance of